### ISSUE TITLE
Instructions need some clarification

### ISSUE THREAD

**Issue by bryanodonoghue** [mangopi-sbc/Tina-Linux#9]
Hello.

I successfully compiled up "Tina-Linux/out/d1-mq_pro/tina_d1-mq_pro_uart0.img"

https://github.com/mangopi-sbc/Tina-Linux

I then subsequently did this

- sudo dd if=tina_d1-mq_pro_uart0.img of=/dev/sdh bs=1M status=progress conv=fdatasync
- Insert the SD into the MangoPI
- Insert a mini HDMI cable
- Power on

No output on the screen.

- Is it possible to use "dd" on this image directly ?
- Should there be any output on the HDMI ?
- Should the UART be hooked up to a USB adapter ?

I'm guessing the output is UART only but, it would be nice to have some statements in the documentation on the above steps, maybe calling out the steps for dd and whether or not a USB-UART adapter is required.

**Comment by mangobuge** [comment]
tina-linux only support allwinner's phoenixcard or phoenixsuit software.

**Comment by arkadijs** [comment]
Converting the image with [OpenixCard](https://github.com/YuzukiTsuru/OpenixCard) results in the Tina-Linux logo on HDMI display. Do I need UART connection to access command line?

**Comment by bryanodonoghue** [comment]
I booted the Armbian image.
No UART was required.

**Comment by vladimir-dudnik** [comment]
Unfortunately Armbian build does not initialize RISCV vector extensions, available for this SoC, so part of CPU functionality is not available

**Comment by bryanodonoghue** [comment]
Armbian runs DOG slow I wouldn't recommend actually using it
We need to get a proper yocto build going

**Comment by vladimir-dudnik** [comment]
What is DOG, which you mention is slow?

**Comment by bryanodonoghue** [comment]
Slow yes sorry "dog slow" means slow

**Comment by vladimir-dudnik** [comment]
Well, I did try Armbian image on mangopi mq pro v1.4, and in console mode its performance looks acceptable. Nevertheless, what is you recomendation?
I'm pretty fine with Tina Linux if it can support network connectivity through WiFi and remote access through ssh out of the box. (Ability to display console on HDMI display could be a plus but is not critical)

**Comment by bryanodonoghue** [comment]
IDK - the desktop seemed slow to me. If Armbian isn't initalising the RISCV vector stuff, that is surely down to kernel or earlier stage firmware so switching user-space environments won't fix it, only a firmware/kernel update will.

**Comment by arkadijs** [comment]
D1 CPU is L1 cache only of 32K+32K.

**Comment by vladimir-dudnik** [comment]
@bryanodonoghue yes, that's right, rvv support should be enabled at linux kernel side. Tina Linux does this and RVBoards Debian does this too, so I thought it could be enabled at Armbian too but seems noone bothers with this.
